src/plugins/resolver/filename.c

```diff
@@ -590,7 +650,7 @@ static int elektraResolveMapperSystem (resolverHandle * p, Key * warningsKey)
  *         be found
  * warnings will be reported to warningsKey
  */
-int ELEKTRA_PLUGIN_FUNCTION (resolver, filename) (Key * forKey, resolverHandle * p, Key * warningsKey)
+int ELEKTRA_PLUGIN_FUNCTION (resolver, filename)(Key * forKey, resolverHandle * p, Key * warningsKey)
```

Comment:
there seem to be plenty of whitespace changes, please rerun scripts/reformat-source